src/common/image.c

```diff
@@ -272,7 +272,7 @@ void dt_image_path_append_version(int imgid, char *pathname, size_t pathname_len
 
 void dt_image_print_exif(const dt_image_t *img, char *line, size_t line_len)
 {
-  if(img->exif_exposure >= 0.1f)
+  if(img->exif_exposure >= 1.0f)
     snprintf(line, line_len, "%.1f'' f/%.1f %dmm iso %d", img->exif_exposure, img->exif_aperture,
```

Comment:
Or:
https://en.wikipedia.org/wiki/Prime_(symbol)
The double prime (″) represents inches (in), arcseconds (as), and seconds (s). However, for convenience, a (") (double quotation mark) is commonly used.